python/ray/rllib/agents/agent.py

```diff
@@ -443,6 +443,26 @@ def export_policy_model(self, export_dir, policy_id=DEFAULT_POLICY_ID):
         """
         self.local_evaluator.export_policy_model(export_dir, policy_id)
 
+    def export_policy_checkpoint(self,
+                                 export_dir,
+                                 filename_prefix="model",
+                                 policy_id=DEFAULT_POLICY_ID):
+        """Export policy model checkpoint to local directory.
```

Comment:
This is TF specific right? Should it say "Export a TF model checkpoint..."?

Reply:
> This is TF specific right? Should it say "Export a TF model checkpoint..."?

I changed the comment. Currently only tensorflow implementation is provided. I googled a little bit and it seems that pytorch has similar concepts. I can help if further pytorch policy graph need to implement this.

Reply:
Got it, makes sense